### DISCUSSION TITLE
第2节课作业(12班)

### DISCUSSION THREAD

**Discussion by fanqiNO1** [InternLM/Tutorial#88] [作业]
12班第2节课的作业交在这里

**Comment by jujimeizuo** [comment]
https://note.jujimeizuo.cn/llm/internlm/lec2/

**Comment by HuangXinzhe** [comment]
https://blog.csdn.net/qq_29787929/article/details/135434363

**Comment by JiangXiaobai00** [comment]
https://d0elb2210mg.feishu.cn/docx/Dwird808nocVb7xrHzmcGPZGnTh?from=from_copylink

**Comment by canxingliuying** [comment]
https://blog.csdn.net/weixin_42449385/article/details/135445393?spm=1001.2014.3001.5502

**Comment by ChengaoJ** [comment]
https://blog.csdn.net/weixin_43193528/article/details/135447287

**Comment by Jeff-Huang-SHU** [comment]
https://blog.csdn.net/2301_77209094/article/details/135446253?spm=1001.2014.3001.5501

**Comment by Smiling-Weeping-zhr** [comment]
https://www.cnblogs.com/smiling-weeping-zhr/p/17952161

**Comment by cheng221** [comment]
https://spice-spice-829.notion.site/1a1dc7130573423eaccd3c58c566e2b0?pvs=4

**Comment by Silflame** [comment]
https://www.yuque.com/wayne-o8axv/hdcch1/yxmxbcbeahns1ii3

**Comment by hxy-tsy** [comment]
[第二节课作业.md](https://github.com/InternLM/tutorial/files/13871418/default.md)

**Comment by sunhanyu714** [comment]
https://github.com/sunhanyu714/Internlm_learning/blob/main/%E7%AC%AC%E4%BA%8C%E8%8A%82%E8%AF%BE%E4%BD%9C%E4%B8%9A.md

**Comment by XiLinky** [comment]
https://blog.csdn.net/m0_73202283/article/details/135482490
笔记+作业截图

**Comment by jiangzf93** [comment]
https://blog.csdn.net/weixin_42890306/article/details/135489258

**Comment by al2h36o30s3** [comment]
https://zhuanlan.zhihu.com/p/677120894

**Comment by deseijily** [comment]
https://j1b8l2ps5tm.feishu.cn/docx/RxUDdRlz1o5XLexF6SycRaOjnJe?from=from_copylink

**Comment by 0-yy-0** [comment]
https://blog.csdn.net/qq_34354951/article/details/135468964

**Comment by ShenFeilang** [comment]
https://ink-neon-bce.notion.site/InternLM-79809d1fd1914b119ccd5efbffc8a37e?pvs=4
